rasa/core/brokers/broker.py

```diff
@@ -34,6 +34,7 @@ async def create(
             sqlalchemy.exc.OperationalError,
             aio_pika.exceptions.AMQPConnectionError,
             aiormq.exceptions.ChannelNotFoundEntity,
+            *aio_pika.exceptions.CONNECTION_EXCEPTIONS,
```

Comment:
Are some of the errors above included in this?

Reply:
I've just double-checked, the errors above are not included in `aio_pika.exceptions.CONNECTION_EXCEPTIONS`.